src/test/groovy/graphql/util/TraverserTest.groovy

```diff
@@ -0,0 +1,107 @@
+/*
+ * Created: Jan 29, 2018 11:56:36 AM
+ * Author: gkesler
```

Comment:
Could you please remove this comment: we don't maintain such references to this original author as comments.

Reply:
sure, sorry about that. that's my IDE template